Source/Scene/Cesium3DTileStyle.js

```diff
@@ -1657,32 +1657,33 @@ Cesium3DTileStyle.prototype.getColorShaderFunction = function (
   }
 
   this._colorShaderFunctionReady = true;
-  this._colorShaderFunction = defined(this.color)
-    ? this.color.getShaderFunction(
-        functionName,
-        propertyNameMap,
-        shaderState,
-        "vec4"
-      )
-    : undefined;
+  this._colorShaderFunction =
+    defined(this.color) && defined(this.color.getShaderFunction)
+      ? this.color.getShaderFunction(
+          functionHeader,
+          variableSubstitutionMap,
+          shaderState,
+          "vec4"
+        )
+      : undefined;
```

Comment:
For readability, I'd rewrite this as an explicit if statement rather than a ternary operator.